packages/fxa-auth-server/lib/bounces.js

```diff
@@ -71,6 +71,7 @@ module.exports = (config, db) => {
         tally.count++;
       }
     });
+    return tallies;
```

Comment:
I think we ought to return the tallies here because the caller in `checkBounces` returns there and it would be more consistent (and less of a footgun) if we either removed the return or added one here - I opted for the latter, but I can change it.

I can also move this part of the diff to a separate PR since it's no longer required as part of this patch now.

Reply:
Couldn't hurt but nothing looks at that value.